mypy/plugins/enums.py

```diff
@@ -87,6 +88,17 @@ def _infer_value_type_with_auto_fallback(
     if proper_type is None:
         return None
     proper_type = get_proper_type(fixup_partial_type(proper_type))
+    # Enums in stubs may have ... instead of actual values. If `_value_` is annotated
+    # (manually or inherited from IntEnum, for example), it is a more reasonable guess
+    # than literal ellipsis type.
+    if isinstance(proper_type, Instance) and proper_type.type.fullname in ELLIPSIS_TYPE_NAMES:
```

Comment:
Is there a way to check this is in a stub file instead of always applying this behavior? I think there's some attribute like "mypy-only" on definition, but maybe I'm misremembering...

Reply:
Not on the definition itself, but we can trace it to the defining module. Done!